src/rules/memberOrderingRule.ts

```diff
@@ -309,12 +321,41 @@ export class MemberOrderingWalker extends Lint.RuleWalker {
                     `Instead, this should come ${locationHint}.`;
                 this.addFailureAtNode(member, errorLine1);
             } else {
+                if (this.opts.alphabetize && member.name) {
+                    if (rank !== prevRank) {
+                        // No alphabetical ordering between different ranks
+                        prevName = undefined;
+                    }
+
+                    const curName = nameString(member.name);
+                    if (prevName !== undefined && curName < prevName) {
```

Comment:
For example, in `ordered-imports`, we use `<` and `>`, but it calls out case-insensitive, lowercase-first and lowercase-last. I think we could just default to case insensitive and call `.toLowerCase()` before comparison